idl/sprot.idol

```diff
@@ -40,14 +40,22 @@ Interface(
                 err: CLike("SprotError"),
             ),
         ),
-    "status": (
-        doc: "Return untrusted status from RoT",
-        reply: Result(
-            ok: "Status",
-            err: CLike("SprotError"),
+        "status": (
+            doc: "Return untrusted status from RoT",
+            reply : Result(
+              ok: "SprotStatus",
+              err: CLike("SprotError"),
+            ),
+            encoding: Hubpack
+		),
```

Comment:
Nit - Indentation is off here; maybe tabs vs spaces? Where is my `idolfmt`

Reply:
Fixed.